tests_e2e/scenarios/runbooks/daily.yml

```diff
@@ -31,6 +25,14 @@ variable:
     value: ""
     is_secret: true
 
+  # The image and location are set by the combinator
+  - name: marketplace_image
+    value: ""
+  - name: location
+    value: ""
+  - name: default_location
+    value: "westus2"
+
```

Comment:
Now that those variables are generated by the combinator, overriding them from the command line has no effect. In future PRs I will replace those in favor of using JSON files defining the test suites, as we discussed earlier.